docs/ios/open-source/manual-instrumentation/logging.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Logging
+description: Capture log messages with different severity levels in your iOS app
+sidebar_position: 6
+---
+
+# Logging
+
+Embrace's logging capabilities allow you to capture log messages with different severity levels, providing valuable context for troubleshooting and debugging issues in your app.
+
+## Understanding Logging in Embrace
+
+Logs in Embrace serve several purposes:
```

Comment:
would note here that they open an immediate HTTP request (which competes with other operations for device bandwidth and resources) as opposed to breadcrumbs which are part of the session payload. Consequently, i'm not sure the below examples are all places we'd recommend logs, given the performance hit.